velox/docs/functions/json.rst

```diff
@@ -89,18 +89,6 @@ special characters unescaped and double quotes removed.
 JSON Functions
 --------------
 
-.. function:: json_extract_scalar(json, json_path) -> varchar
```

Comment:
Thank you for the cleanup. Since this is an unrelated change, would you submit it as a separate PR?